string name
float32 angular_velocity
